### DISCUSSION TITLE
Top icon 1-3 usable as chips

### DISCUSSION THREAD

**Discussion by denes44** [Blackymas/NSPanel_HA_Blueprint#730] [Ideas]
Hi,

I think it would be good if the first 3 icons would be converted to chips as icons 4-10.
For example I don't and never will use the relays of the panel, so it would be nice to use those icons as chips as well.
And you can always setup the chips to behave like the first two icons, if somebody wants to keep this feature.

**Comment by edwardtfn** [comment]
- #1725
